gci/models.py

```diff
@@ -1 +1,27 @@
-# Create your models here.
+import json
+from django.db import models
+
+
+class Students(models.Model):
+    id = models.IntegerField(default=0, primary_key=True)
```

Comment:
Please rename this field to `identifier` instead. `id` would interfere with python's `built-in` function.